extension/src/popup/components/sendPayment/SendConfirm/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import StellarSdk, { Asset } from "stellar-sdk";
+import { Types } from "@stellar/wallet-sdk";
+
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import {
+  signFreighterTransaction,
+  submitFreighterTransaction,
+  ActionStatus,
+  transactionSubmissionSelector,
+} from "popup/ducks/internalTransaction";
+import { AppDispatch } from "popup/App";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+
+import { PopupWrapper } from "popup/basics/PopupWrapper";
+import { BackButton } from "popup/basics/BackButton";
+
+import { SubmitFail, SubmitPending, SubmitSuccess } from "./SubmitResult";
+
+import "../styles.scss";
+
+export const SendConfirm = ({
+  publicKey,
+  amount,
+  asset,
+  destination,
+  transactionFee,
+  memo,
+}: {
+  publicKey: string;
+  amount: string;
+  asset: string;
+  destination: string;
+  transactionFee: string;
+  memo: string;
+}) => {
+  const dispatch: AppDispatch = useDispatch();
+  const networkDetails = useSelector(settingsNetworkDetailsSelector);
+  const submission = useSelector(transactionSubmissionSelector);
+  const [isViewOnly, setIsViewOnly] = useState(false);
```

Comment:
Hm, can we call this `isViewingDetails`? I was a little confused by what this meant

Reply:
I changed it to `isSendComplete`, I think having "view" in there makes it confusing since you're viewing either way. If you think of a better name though lmk and I can change it